apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -418,28 +418,28 @@ suite('ActivityHandler', function() {
 
       test('email in notification title when contact without name', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.title, emailAddress);
+        var notification = sendSpy.firstCall;
+        assert.isTrue(notification.calledWith(emailAddress));
       });
     });
 
     suite('after getSelf', function() {
       var sendSpy, contactSpy;
       setup(function(done) {
-        sendSpy = this.sinon.spy(window, 'Notification');
+        sendSpy = this.sinon.spy(NotificationHelper, 'send');
         contactSpy = this.sinon.spy(Contacts, 'findByAddress');
         MockNavigatormozApps.mTriggerLastRequestSuccess().then(done, done);
       });
 
       test('a notification is sent', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.body, message.body);
-        assert.deepEqual(notification.data, {
+        var notification = sendSpy.firstCall;
+        assert.equal(notification.args[1].body, message.body);
+        assert.deepEqual(notification.args[1].data, {
           id: message.id,
           threadId: message.threadId
         });
-        assert.equal(notification.icon, 'sms');
+        assert.equal(notification.args[1].icon, 'sms');
```

Comment:
calledWith and calledWithMatch allow you to skip remaining arguments, but if you want to skip the first ones, you can use sinon.match.any. See all predefined values here: http://sinonjs.org/docs/#sinon-match-api

Matching allows you to put only some of the properties. I think that for objects if you don't use match you need a reference equality, while with match it compares the properties values ... to be checked :)